### PR TITLE
Vl cart quantity

### PR DESCRIPTION
**Pull request by vinhle000**
Added dropdown for selecting Quantity for the product.

Used a react selector to populate the options based on the Quantity value  which was determined by the Size selected prior.

**Ticket:** https://trello.com/c/l3FGeEKU

**Behavior:**
AddToCart_QuantitySelector

### CODE REVIEW

File: src/components/overview/addToCart/AddToCart.jsx [hunk 1]
```diff
@@ -12,46 +12,75 @@ const AddToCart = (props) => {
...
   let styleSelector;
-  if (Object.entries(currentSkus).length > 0){
 
-    styleSelector =
-             ( <select value={selectedSize} onChange={handleSizeSelect}>
+  if (Object.entries(currentSkus).length > 0){
+    styleSelector = (
+             <select name={selectedSize} onChange={handleSizeSelect}>
 
                 {selectedSize === '' && <option>Select Style</option>}
```
Comment (ygroener) [inline src/components/overview/addToCart/AddToCart.jsx:35]: Should the Size selector say 'Select Size'? Right now it's labeled select style

**Review by ygroener — CHANGES_REQUESTED** [review]
Looks good! Just added the one change I saw

File: src/components/overview/addToCart/AddToCart.jsx (hunk 1 above)
Comment (vinhle000) [reply src/components/overview/addToCart/AddToCart.jsx:35]: Thanks for catching that. Pushed up the changes.

**Review by ygroener — APPROVED** [review]
Looks good!

**Pull request merged**